### PR TITLE
A complete website with navigation

### PR DESCRIPTION
**Pull request by gersonhiggins**
Add header and footer.
Add Contact Me info.
Add Styles.
Add JS for Menu Navigation.
Check and Fix Linters.

### CODE REVIEW

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @gersonhiggins and @benja27 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      node4.setAttribute('data-id', `${index}`);
+      node1.appendChild(node4);
+    });
+  }
+
+  removeBook(index) {
+    this.booksArr.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(this.booksArr));
+    this.displayBooks();
+  }
```
Comment (didierganthier) [inline scripts/main.js:83]: [OPTIONAL] Well done overall 💯  but consider separating functionality into smaller methods:
The `addBook`, `displayBooks`, and `removeBook` methods all do different things. Consider separating the functionality into smaller, more specific methods that are responsible for a single task. ✔️  For example, you could create a `createBookNode` method that is responsible for creating the HTML node for a single book, and then call that method from within the `displayBooks` method. 💡

File: scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      const suffix = suffixes[day - 1];
+      const year = date.getFullYear();
+      const hour = date.getHours() % 12 || 12;
+      const minute = date.getMinutes();
+      const second = date.getSeconds();
+      const ampm = date.getHours() >= 12 ? 'pm' : 'am';
+
+      const formattedDate = `${month} ${day}${suffix} ${year}, ${hour}:${minute}:${second} ${ampm}`;
+
+      this.timeContainer.innerHTML = formattedDate;
```
Comment (didierganthier) [inline scripts/main.js:41]: [OPTIONAL] Nice work on the date 📆 but avoid using `innerHTML` for security reasons:
The `innerHTML` property can be a security risk if the content being inserted is not properly sanitized. Consider using alternative methods like `textContent` or `createElement` instead. 👍🏽

**Review by didierganthier — APPROVED** [review]
Hi @gersonhiggins and @benja27 ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Navigation working ✔️ 
- Clean JavaSript code ✔️ 
- Good DOM interactions ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**